### PR TITLE
kie-issues#296: Fix CVE-2022-3517 in kogito-apps and kie-tools

### LINKED ISSUE #296: Fix `CVE-2022-3517` in kogito-apps and kie-tools
Issue: [CVE-2022-3517](https://www.mend.io/vulnerability-database/CVE-2022-3517)

`minimatch@3.0.4` has a vulnerability that's fixed in `3.0.5` versions. Both `kie-tools` & `kogito-apps` are using it and should upgrade it.

This dependency is being used in `kie-tools` in:
- chrome-extension
- editor
- kie-editors-dev-vscode-extension
- kie-editors-standalone
- serverless-logic-web-tools
- serverless-workflow-standalone-editor
- vscode-extension
- workspace-git-fs

This dependency is brought transitively to `kogito-apps` because a dependency to `@kie-tools-core/editor`, this could be addressed by changing the dependency to the latest version of `kie-tools` after `kie-tools` is fixed.

Acceptance criteria
- [ ] Upgrade `minimatch` to a fixed version in `kogito-apps` & `kie-tools`
- [ ] Make `kogito-apps`depend on latest `kie-tools`

### PR DESCRIPTION
**Pull request by pefernan**
Closes https://github.com/kiegroup/kie-issues/issues/296

Fixes `CVE-2022-3517` vulnerability on `minimatch`:`3.0.4`.

This is part of an ensemble, please merge with:
https://github.com/kiegroup/kie-tools/pull/1710
https://github.com/kiegroup/kogito-apps/pull/1746

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -14675,7 +14026,7 @@ packages:
     deprecated: this version had an improper engines field added, update to 1.1.1
     dependencies:
       "@gar/promisify": 1.1.2
-      semver: 7.3.8
+      semver: 7.3.7
```
Comment (jstastny-cz) [inline pnpm-lock.yaml]: This seems like a suspicious downgrading.

**Review by jstastny-cz — COMMENTED** [review]
There is quite some disproportion between removed and added lines, the lock file has whole entries removed, like
`packages/vscode-extension-pack-kogito-kie-editors:`
`packages/vscode-extension-pmml-editor:`

That's kinda surprising, was the lock out of date or did we use e.g. reduced scope during upgrading now?

File: pnpm-lock.yaml (hunk 1 above)
Comment (pefernan) [reply pnpm-lock.yaml]: @jstastny-cz totally... I was surprised about the removed entries in the lock file too. It seems that it was really outdated, looks like those packages where removed but still present in the `.lock`

Comment (pefernan) [reply pnpm-lock.yaml]: @jstastny-cz well looks like `@npmcli/fs@1.1.0` has a depends on `semver@^7.3.5` so both resolutions are right. IDK why pnpm decided to downgrade this.. just upgraded back to `7.3.8` to keep it as it was before.

**Review by tiagobento — APPROVED** [review]

**Review by tiagobento — APPROVED** [review]

**Pull request merged**